.azure-pipelines.yml

```diff
@@ -0,0 +1,27 @@
+# https://github.com/Microsoft/azure-pipelines-image-generation/blob/master/images/macos/
+variables:
+  VM_IMAGE: macOS-10.15
+  XCODE_VERSION: 12.1
+  PLATFORM_VERSION: 14.1
+  DEVICE_NAME: iPhone 11 Pro Max
+  DEFAULT_NODE_VERSION: "12.x"
+
+pool:
+  vmImage: "$(VM_IMAGE)"
+
+stages:
+- stage: Unit_Tests_And_Linters
+  jobs:
+  - job: Node_Unit_Tests
+    steps:
+    - template: azure-templates/node_setup_steps.yml
```

Comment:
Should be `ci-jobs/templates/node_setup_steps.yml`?

Reply:
fixed